src/fty_info_server.c

```diff
@@ -142,7 +142,11 @@ fty_info_server (zsock_t *pipe, void *args)
                  }
                  if (streq(command, "VERSION")) {
                     zmsg_t *reply = zmsg_new ();
+                    // are we on RC?
                     char *version = s_readall ("/etc/bios-release.json");
```

Comment:
Nowadays this is called `/etc/release-details.json` (`*bios*` one is a symlink that may be gone sometime later).
And containers (at least those instantiated by `ci-reset-virtual-machine.sh` should have this file too.